### PR TITLE
directly exports promise

### PR DESCRIPTION
**Pull request by Sinewyk**
keep backwards compatibility
ref https://github.com/stefanpenner/es6-promise/issues/181

### CODE REVIEW

**Comment by Sinewyk** [comment]
edit: remove worker edit and keep global export on window as `ES6Promise`

File: lib/es6-promise.umd.js [hunk 1]
```diff
@@ -1,18 +1,16 @@
 import Promise from './es6-promise/promise';
 import polyfill from './es6-promise/polyfill';
 
-var ES6Promise = {
-  'Promise': Promise,
-  'polyfill': polyfill
-};
+Promise.Promise = Promise;
+Promise.polyfill = polyfill;
```
Comment (stefanpenner) [inline lib/es6-promise.umd.js]: Should we may want to define these as non enumerable (if oO.defineProp exists)?

Comment (Sinewyk) [inline lib/es6-promise.umd.js]: Stole this from bluebird https://github.com/petkaantonov/bluebird/blob/master/src/promise.js#L771

They don't seem to do it so I wouldn't bother with it.

Comment (stefanpenner) [inline lib/es6-promise.umd.js]: okidokies

**Comment by stefanpenner** [comment]
thanks

**Pull request merged**